src/west/commands.py

```diff
@@ -224,6 +226,29 @@ def check_output(self, args, cwd=None):
         log.dbg(f"running '{cmd_str}' in {cwd or os.getcwd()}",
                 level=log.VERBOSE_VERY)
         return subprocess.check_output(args, cwd=cwd)
+
+    def git_version_at_least(self, version):
```

Comment:
minor, should we call it `git_version_minimum_required` / `git_version_minimum`?

Reply:
> minor, should we call it `git_version_minimum_required` / `git_version_minimum`?

I changed it to a property, `git_version_info`, following along with `sys.version_info`.